### DISCUSSION TITLE
rule|text|BR-BT-00137-0210 : "If the procedure is subdivided into lots, then lot IDs must be different from LOT-0003"

### DISCUSSION THREAD

**Discussion by MicheleAmoroso** [OP-TED/eForms-SDK#802] [Q&A] [answered]
Can someone explain this assert to me about this field: **BT-137-LotsGroup** ? I noticed that on eNotices2 lots have incremental id starting from LOT-0001 and do not skip LOT-0003 but from the rule it seems that a LOT-003 should not be present. Can you explain to me better the meaning of the rule mentioned above? Thank you.

**Comment by YvesJo** [comment, marked as answer]
Hi,
The label for the rule is inappropriate and shall soon be corrected. 
The rule actually checks that for any defined Group of Lots, the Group of Lots composition is actually defined (i.e. there exists a field BT-330-Procedure with the same Technical ID as the one in BT-137-LotsGroup).

**Reply by MicheleAmoroso** [reply]
Thank you!
